packages/backend/src/App.ts

```diff
@@ -502,6 +504,7 @@ export default class App {
                 }
                 return 0.2;
             },
+            profilesSampleRate: 0.2, // 20% of samples will be profiled
```

Comment:
> Set sampling rate for profiling - this is relative to tracesSampleRate

So we are going to get 20% of 20%. Double checking this is what we want.

Reply:
yup!